src/github.com/couchbaselabs/sync_gateway/base/bucket.go

```diff
@@ -77,6 +85,23 @@ func (bucket couchbaseBucket) View(ddoc, name string, params map[string]interfac
 }
 
 func (bucket couchbaseBucket) StartTapFeed(args walrus.TapArguments) (walrus.TapFeed, error) {
+	// Use tap only when it's explicitly requested, or if DCP isn't supported
+	if bucket.spec.FeedType == "tap" {
```

Comment:
is it possible to define an enum for the different feed types? I think this control structure might be slightly better as:
```
switch (bucket.spec.Feedtype) {
case TAP:
     // do stuff
case DCP:
     // do other stuff
default: 
     // return an error
} ```